licenses.d2iq.yaml

```diff
@@ -396,7 +396,12 @@ resources:
       - url: https://github.com/thanos-io/thanos
         ref: ${image_tag}
         license_path: LICENSE
-  - container_image: registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.6.0
+  - container_image: k8s.gcr.io/kube-state-metrics/kube-state-metrics:v1.9.8
```

Comment:
Does this change require updating something else (in confulence or third party legal site perhaps)

Reply:
I'm not sure.. @mhrabovcin would you know?